llvm-runtimes/openmp/openmp-18.1.8.ebuild

```diff
@@ -41,8 +41,7 @@ DEPEND="
 BDEPEND="
```

Comment:
But does that actually work? When I've tried with 19.x and 20.x, disabling both plugins resulted in CMake failing over no objects for DeviceRTL.

Reply:
`>=llvm-runtimes/openmp-19` doesn't have `IUSE="offload"`, so there is nothing to change in those versions. I did test with 18 and older, and DeviceRTL is built fine with neither target enabled.